backend/flask_api.py

```diff
@@ -161,7 +161,8 @@ def add_project():
                 return_ids = uploader.save_manifest_to_db(
                     json5.loads(request.data.decode('utf-8')))
             else:
-                raise ApiException("Wrong content header and no files attached", 400)
+                raise ApiException("Wrong content header\
+                 and no files attached", 400)
```

Comment:
pep8 discourages line breaks in statements and encourages (same line) operators instead